src/blog/building-icarus.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Icarus - Taming the Kubernetes Beast'
+slug: building-icarus
+authors: ['pawalt']
+publishedAt: 2020-05-27
+coverPhoto: ../images/blog/icarus.jpg
+---
+
+As of recent, we've moved our infrastructure away from [Dokku](http://dokku.viewdocs.io/dokku/) and onto [Kubernetes](https://kubernetes.io/). Especially for an organization with high turnover (nature of being a club), you have to carefully consider the organizational impacts of moving to a technology as complex as Kubernetes. This post should serve as a bit of insight into how we dealt with that challenge and built technology to solve it.
```

Comment:
I also think this paragraph could be reworded a decent bit -- "Especially for an organization with high turnover (nature of being a club), you have to carefully consider the organizational impacts of moving to a technology as complex as Kubernetes." is kind of a mouthful. Maybe flip it around: "As a student organization [or club], we have to be cognizant of the fact that members turn over all the time. Because of this, we have to carefully consider the organizational impacts of moving to a technology as complex as Kubernetes"

Reply:
Hows this?